packages/website/src/components/playground.html

```diff
@@ -1,91 +1,482 @@
+          </div>
+          <div class="flex-1">
+            <div class="flex flex-col w-full">
+              <ul id="tabs" class="[&>li]:inline-block [&>li]:p-2 [&>li]:border [&>li]:rounded-t [&>li]:border-b-0 [&>li]:border-transparent">
+                <li data-tab="errors" class="tab cursor-pointer !border-gray-200 bg-white">
+                  Errors
+                </li>
+                <li data-tab="fixed" class="tab cursor-pointer">
+                  Fixed
+                </li>
+              </ul>
+            </div>
+            <div class="border border-gray-300 rounded flex-grow p-2 bg-gray-100 h-[400px] overflow-auto">
+              <div id="errors" class="block">
+                <p class="text-gray-500">No errors detected. Try entering some JavaScript code like:</p>
+                <code class="bg-white p-1 rounded text-sm">let a = 1<br>console.log(a)</code>
+              </div>
+              <div id="fixed" class="hidden">
+                <div id="fixed-content" class="text-sm">
+                  <p class="text-gray-500">Fixed code will appear here</p>
+                </div>
+              </div>
+            </div>
           </div>
-                    <pre id="fixed" class="hidden text-sm">
-                        HERE
-                    </pre>
+        </div>
+            
+        <div class="mt-4 flex flex-wrap gap-2" id="playground">
+          <button
+            type="button"
+            id="applyFixBtn"
+            class="bg-blue-600 text-white px-4 py-2 rounded hover:bg-blue-700 transition-colors"
+          >Apply Fix</button>
+          <button
+            type="button"
+            id="applySuggestionBtn"
+            class="bg-green-600 text-white px-4 py-2 rounded hover:bg-green-700 transition-colors"
+          >Apply Suggestion</button>
+          <button
+            type="button"
+            id="analyzeBtn"
+            class="bg-purple-600 text-white px-4 py-2 rounded hover:bg-purple-700 transition-colors"
+          >Analyze Code</button>
+          
+          <div id="statusMessage" class="w-full mt-2 p-2 rounded hidden"></div>
         </div>
       </div>
-    </div>
-    <script type="module" src="~/src/scripts/playground/app.js"></script>
-    <script>
-          const $tabs = document.getElementById("tabs");
-          $tabs.addEventListener("click", (e) => {
-            const $errors = document.getElementById("errors");
-            const $fixed = document.getElementById("fixed");
-
-            if (e.target) {
-              let $openTarget = null;
-              let $closeTarget = null;
-
-              if (e.target.dataset.tab === "errors") {
-                $openTarget = $errors;
-                $closeTarget = $fixed;
-              } else if (e.target.dataset.tab === "fixed") {
-                $openTarget = $fixed;
-                $closeTarget = $errors;
-              }
-              if ($openTarget || $closeTarget) {
+        
+      <script type="module">
```

Comment:
Could you move the JavaScript code to `packages/website/src/scripts/playground/app.js` and remove the inline script?